docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -705,7 +705,8 @@ AWS API, the logs will show the error message stack trace.
 
 Within the logs you may see a connection failure such as a i/o timeout
 regarding `sts.amazonaws.com:443`. The Teleport Application Service requires
-connecting to `https://sts.amazonaws.com` to create an authorized AWS console session.
+connecting to `https://sts.amazonaws.com` and `https://signin.aws.amazon.com/federation`
```

Comment:
s/requires connecting to/must be able to connect to/g

Reply:
thanks @zmb3 , please see update